db/migrate/20190110201414_add_multiattach_to_cloud_volume.rb

```diff
@@ -0,0 +1,5 @@
+class AddMultiattachToCloudVolume < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_volumes, :multiattach, :boolean
```

Comment:
Hm `multiattach` looks overly shortened when it doesn't need to be, how about `multi_attach` or `multiple_attachment`?

Reply:
Correct, but it matches to OpenStack keywords - https://docs.openstack.org/cinder/latest/admin/blockstorage-volume-multiattach.html

Reply:
True but this is provider common code so matching any specific provider isn't as important as it matching the rest of the platform consistently.

Reply:
@agrare multi_attach or multiple_attachment would suit me fine, do you have a preference?

Reply:
@mansam I think `multi_attachment` would be good, I'm bad at naming :)